src/Microsoft.Azure.ServiceBus/Core/MessageReceiver.cs

```diff
@@ -202,6 +202,14 @@ internal set
         public virtual string Path { get; private set; }
 
         /// <summary>
+        /// Duration after which individual operations will timeout.
```

Comment:
It looks like this tag isn't natively supported by C#. Do I need additional libraries to get this working? I tried using the tag and VS intellisense doesn't show the comment. Am I missing something?